src/fe-small-advertisements/package.json

```diff
@@ -19,7 +19,7 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "react-scripts test"
+    "lint": "eslint '*/**/*.{js,ts,tsx}' --config ./.eslintrc.js"
   },
   "eslintConfig": {
```

Comment:
Can you also delete this section from here?

Reply:
Done